google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/AddCustomerMatchUserList.java

```diff
@@ -219,11 +222,13 @@ private void addUsersToCustomerMatchUserList(
       // NOTE: The details of each partial failure error are not printed here, you can refer to
       // the example HandlePartialFailure.java to learn more.
       if (response.hasPartialFailureError()) {
+        GoogleAdsFailure googleAdsFailure = ErrorUtils.getInstance()
+            .getGoogleAdsFailure(response.getPartialFailureError().getDetails(0));
         System.out.printf(
             "Encountered %d partial failure errors while adding %d operations to the offline user "
                 + "data job: '%s'. Only the successfully added operations will be executed when "
                 + "the job runs.%n",
-            response.getPartialFailureError().getDetailsCount(),
+            googleAdsFailure.getErrorsCount(),
```

Comment:
There are other examples that work with partial failure errors. Do those need to be updated as well?

```
com/google/ads/googleads/examples/errorhandling/HandlePartialFailure.java
com/google/ads/googleads/examples/remarketing/AddCustomerMatchUserList.java
com/google/ads/googleads/examples/remarketing/UploadCallConversion.java
com/google/ads/googleads/examples/remarketing/UploadConversionWithIdentifiers.java
com/google/ads/googleads/examples/remarketing/UploadStoreSalesTransactions.java
com/google/ads/googleads/examples/remarketing/UploadConversionAdjustment.java
com/google/ads/googleads/examples/remarketing/UploadConversionEnhancement.java
com/google/ads/googleads/examples/remarketing/UploadOfflineConversion.java
```

Reply:
Good point. I've updated those as well.

I've also made some improvements to the ErrorUtils class:
- rethrow InvalidProtocolBufferException as an unchecked exception 
- add convenience method to get GoogleAdsFailure from Status.
- add convenience method to get failed operation indices from GoogleAdsErrors.